service/src/test/kotlin/io/provenance/digitalcurrency/consortium/stream/RpcEventStreamTest.kt

```diff
@@ -0,0 +1,28 @@
+package io.provenance.digitalcurrency.consortium.stream
+
+import io.provenance.digitalcurrency.consortium.TestContainer
+import io.provenance.digitalcurrency.consortium.config.logger
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+
+@TestContainer
+internal class RpcEventStreamTest {
+    @Autowired
+    private lateinit var eventStreamFactory: EventStreamFactory
+    private lateinit var rpcEventStream: RpcEventStream
+
+    @BeforeAll
+    fun beforeAll() {
+        rpcEventStream = eventStreamFactory.getStream(
+            eventTypes = listOf(),
+            startHeight = 0,
+            observer = EventStreamResponseObserver { logger().info("test") }
+        )
+    }
+
+    @Test
+    fun streamEvents() {
+        rpcEventStream.streamEvents()
```

Comment:
@ranadewall was there a plan to add some tests to this?

Reply:
yes there is a plan to add all the tests when I can come up for air.

Reply:
im gonna delete this one for now to get this pr out the door since its not critical

Reply:
go ahead. It's on my list to address code coverage here soon.